04-loop.md

```diff
@@ -521,4 +525,4 @@ so she decides to get some coffee and catch up on her reading.
 >     done
 > done
 > ~~~
-
+> This is a test
```

Comment:
Thanks Ashwin. Deleted the line:

> +> This is a test

Cheers,

Dr Francis Gacenga
QCIF eResearch Analyst
Office of Research Development | Research and Innovation Division | University of Southern
Queensland | Toowoomba, Queensland | 4350| Australia
Phone: +61 7 4687 5640 | 0477 072 707 | Email: gacenga@usq.edu.au<mailto:gacenga@usq.edu.au> | Twitter: https://twitter.com/USQ_eResearch
Website: https://www.usq.edu.au/eresearch |QRIScloud: https://www.qriscloud.org.au/

From: Ashwin Srinath [mailto:notifications@github.com]
Sent: Monday, 23 May 2016 8:47 PM
To: swcarpentry/shell-novice <shell-novice@noreply.github.com>
Cc: Francis Gacenga <Gacenga@usq.edu.au>; Mention <mention@noreply.github.com>
Subject: Re: [swcarpentry/shell-novice] Update 04-Loop.md (#366)


In 04-loop.md<https://github.com/swcarpentry/shell-novice/pull/366#discussion_r64202335>:

> @@ -521,4 +525,4 @@ so she decides to get some coffee and catch up on her reading.

>  >     done

>  > done

>  > ~~~

> -

> +> This is a test

Please delete this line

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/swcarpentry/shell-novice/pull/366/files/517d6fd3fe0e4ed43e7eb7ec3d00969ec6922006#r64202335>


_____________________________________________________________
This email (including any attached files) is confidential and is for the intended recipient(s) only. If you received this email by mistake, please, as a courtesy, tell the sender, then delete this email.

The views and opinions are the originator's and do not necessarily reflect those of the University of Southern Queensland. Although all reasonable precautions were taken to ensure that this email contained no viruses at the time it was sent we accept no liability for any losses arising from its receipt.

The University of Southern Queensland is a registered provider of education with the Australian Government.
(CRICOS Institution Code QLD 00244B / NSW 02225M, TEQSA PRV12081 )